src/jet_server.cpp

```diff
@@ -57,56 +66,88 @@ void JetServer::addJetState(const std::string& path)
                             int64_t oldValue = component.getPropertyValue(propertyName);
                             int64_t newValue = value.get(propertyName, "").asInt64(); 
                             if (oldValue != newValue)
+                            {
                                 component.setPropertyValue(propertyName, newValue);
-                            else std::cout << "Value for " << propertyName << " has not changed. Skipping.." << std::endl;
+                                atLeastOnePropertyChanged = true;
+                            }
+                            else 
+                                std::cout << "Value for " << propertyName << " has not changed. Skipping.." << std::endl;
```

Comment:
Please remove: std::cout. It is nice for debugging but should not be in a clean solution. Use the Logger from openDAQ SDK. But for me this line needs not to be logged at all.